frontend/src/metabase/public/containers/PublicOrEmbeddedQuestion/PublicOrEmbeddedQuestionView.tsx

```diff
@@ -78,8 +78,6 @@ export function PublicOrEmbeddedQuestionView({
         )}
         question={question}
         result={result}
-        uuid={uuid}
-        token={token}
```

Comment:
Why don't these 2 props be removed from the component and also from passing to `<Visualization />`?

https://github.com/metabase/metabase/blob/73a25b84ef475dd4c6c44a705f54b5ed9d0d1386/frontend/src/metabase/public/containers/PublicOrEmbeddedQuestion/PublicOrEmbeddedQuestionView.tsx#L143-L144

Reply:
I added a commit that uses only context for it, but let's see if tests pass. Because for the dashboard we need also to wrap each <Visualization> in a dashcard inside a context as well